server/middleware/trimInput.ts

```diff
@@ -0,0 +1,17 @@
+import { RequestHandler } from 'express'
+
+const trimStringValues = (body?: Record<string, unknown>) =>
```

Comment:
Not sure the body param needs to be optional.

Reply:
You might be right actually -- the Express docs specify that, [by default, `req.body` is `undefined`](https://expressjs.com/en/api.html#req.body), but logging both `req.body` and `req.query` on a page with no values expected does return `{}` for both. This might need to be updated, then!